pom.xml

```diff
@@ -344,6 +344,48 @@
                     </execution>
                 </executions>
             </plugin>
+
+            <!-- License stuff -->
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>license-maven-plugin</artifactId>
+                <version>1.8</version>
+                <inherited>false</inherited>
+                <configuration>
+                    <excludedScopes>test</excludedScopes>
+                    <excludedGroups>${project.groupId}</excludedGroups>
+                </configuration>
+                <executions>
+                    <execution>
+                        <id>download-licenses</id>
+                        <goals>
+                            <goal>download-licenses</goal>
+                        </goals>
+                        <phase>generate-resources</phase>
```

Comment:
This needs to bind to a phase later than package otherwise the goal will use the "installed" package which usually isn't there. `verify` is probably our best bet since it's post package but pre-install (since there isn't `post-package`)